rust/repositories.bzl

```
@@ -441,6 +460,10 @@ def _rust_toolchain_repository_impl(ctx):
     if ctx.attr.rustfmt_version:
         BUILD_components.append(_load_rustfmt(ctx))
 
+    # Nightly Rust builds after 2020-05-22 need the llvm-tools gzip to get the libLLVM dylib
+    if ctx.attr.version == "nightly" and int(ctx.attr.iso_date.replace("-", "")) > 20200522:
```

There is no need to convert to int, the alphabetical ordering should just work:

```suggestion
    if ctx.attr.version == "nightly" and ctx.attr.iso_date > "2020-05-22":
```

This is more readable and both operations are `O(len(ctx.attr.iso_date))` anyway

---

Interesting! Didn't realize that would actually work, but it makes sense.